Photon-Tinker/Mesh/MeshSetupFlowUIManager.swift

```diff
@@ -90,10 +84,25 @@ class MeshSetupFlowUIManager : UIViewController, Storyboardable, MeshSetupFlowMa
         log("sticker found by user")
 
         DispatchQueue.main.async {
-            let scanVC = MeshSetupScanStickerViewController.loadedViewController()
-            scanVC.setup(didFindStickerCode: self.setTargetDeviceStickerString, deviceType: self.targetDeviceType)
-            self.embededNavigationController.pushViewController(scanVC, animated: true)
+            if (!self.rewindTo(MeshSetupScanStickerViewController.self)) {
```

Comment:
What's the new `rewindTo` pattern? Is it invoked when the user goes forward after having clicked back so that the target view controller is already in `self.embededNavigationController.viewControllers`? If so, rewind is not the best name for this method. `forwardTo` would describe this better.

Reply:
It is the opposite of what you are assuming. The navigation has the stack of view controllers, but it is known that there are never 2 instances of the same view controller in stack. When setup flow requests to display the certain view controller, the UI manager doesn't know if this is happening because flow has advanced forward or stepped back. For this reason, UI manager tries to look into the stack of views it has and if there's a view of the requested type, it means user is navigating back, and UI manager will uncover the requested view by removing views stacked on top of it. If the view is not present on the stack, it means user is stepping forward and UI manager will push the view on-to the stack.

Reply:
I see. So `self.embededNavigationController.popToViewController` will throw away all the other VCs stacked on top of the target so `rewindTo` will return always false when going forward.

Reply:
Yes. That returned boolean indicates whether rewind attempt was successful which will only be successful when going back in history stack.